### DISCUSSION TITLE
What is advantage of next.js than react.js

### DISCUSSION THREAD

**Discussion by galaxydev55** [Web-development-s/Stripe-with-laravel#125] [Ideas]
I think react.js is the base javascript frameworks

**Comment by Fire-David116** [comment]
Compared to React, Next. js reduces the amount of time an engineer needs to wait for code to refresh, minimizing developer frustration and slowdowns. Data fetching and load times: Next. js can traverse the React tree and query for data in the server, allowing for pre-loaded page data.

**Reply by Fire-David116** [reply]
I think so

**Reply by galaxydev55** [reply]
Wonderful answer
Thanks for your effort
